Gruntfile.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+var seleniumVersion = '2.45.0',
+    seleniumMinorVersion = seleniumVersion.split('.').slice(0, 2).join('.');
+
+process.env.PATH = 'node_modules/.bin/:' + process.env.PATH;
+
+module.exports = function (grunt) {
+    require('load-grunt-tasks')(grunt);
+    require('./test/tasks/appium')(grunt);
+
+    function setBrowser(browserName) {
+        if (grunt.option('browser')) {
+            browserName = grunt.option('browser');
+        }
+        process.env._BROWSER = browserName;
+        grunt.config.set('start-selenium-server.dev.options.serverOptions.browser', browserName);
+    }
+
+    function setEnv(env) {
+        var testSrc = 'mochaTest.test.src',
+            specDir = env.match(/^(functional|multibrowser)$/);
+
+        process.env._ENV = env;
+
+        if (!grunt.config.get(testSrc)) {
+            if (specDir) {
+                // only test functional test spec if required
+                grunt.config.set(testSrc, ['test/spec/' + specDir[0] + '/**/*.js']);
+            } else {
+                // otherwise test global + device specific test specs
+                grunt.config.set(testSrc, ['test/spec/' + env + '/**/*.js', 'test/spec/*.js']);
+            }
+        }
+    }
+
+    //TODO istanbul code coverage
+    // "coverage": "./node_modules/.bin/istanbul cover -x \"**/helpers/_*.js\" ./test/runner.js",
```

Comment:
mhm .. I can't find it in `/test/runner.js`. We need to make sure that istanbul get triggered if we run our tests in travis